build.gradle.kts

```diff
@@ -67,6 +68,16 @@ tasks {
         targetCompatibility = "11"
         options.compilerArgs.addAll(exportsArgs)
         options.errorprone.disableWarningsInGeneratedCode.set(true)
+
+        // enforce general checks
+        options.errorprone.check("BadImport", CheckSeverity.ERROR)
+        options.errorprone.check("MixedMutabilityReturnType", CheckSeverity.ERROR)
+
+        // enforce errorprone-specific checks
+        options.errorprone.check("MemoizeConstantVisitorStateLookups", CheckSeverity.ERROR)
+        options.errorprone.check("ASTHelpersSuggestions", CheckSeverity.ERROR)
+        // our bugpattern classes use custom names
+        options.errorprone.check("BugPatternNaming", CheckSeverity.OFF)
```

Comment:
these changes address/silence all build warnings that we get on the main CI right now:
https://github.com/KengoTODA/errorprone-slf4j/actions/runs/3246218598/jobs/5324715475

WDYT should we enforce the `BugPatternNaming` check as well and rename our classes?
for example `DoNotPublishSlf4jLogger.java` -> `Slf4jLoggerShouldBePrivate.java`

Reply:
Yes it's better, could you raise a separated PR to do it?
Thanks for your contribution!

Reply:
ok raised a PR: https://github.com/KengoTODA/errorprone-slf4j/pull/242